docs/design-proposals/cloud-edge-communication/cloud-edge-communication-framework.md

```diff
@@ -11,18 +11,255 @@ Because the cloud and edge is located in the network boundary of different secur
 However, there may be many problems when VPN is used to directly connect the components/services of different security levels:
 
 * It is too risky to directly expose the central cloud management network/IP to users
-
 * IP needs to be planned globally in advance to ensure that the management components/service IP of the central cloud and each edge site do not conflict
-
 * Network changes can be difficult
 
 This proposal aims to outline a design for cloud-edge communication framework/service to solve the above problems
 
 ## Goals
 
 * Support communication access capability of cloud-edge basic management components
-
 * Support independent IP addressing and IP address convergence of edge management components/services
-
 * Minimize attack surface, zero trust security
 
+## Proposal
+
+The following two solutions do not conflict with each other. The former can converge IP addresses to resolve IP address conflicts. However, the communication needs to be first sent to the gateway and then to the destination service. The latter cannot converge IP addresses but can resolve IP address conflicts. The latter provides end-to-end communication.
```

Comment:
What does "converge IP addresses" here mean?

Reply:
It means expose only the proxy（or gateway/hub） IP address to the cloud, not all internal IP addresses of the site. The advantage of this method is that the firewall does not need to be configured with a large number of rules and does not need to be frequently changed due to the changes of the edge site network. Converge IP addresses can help minimize the attack surface.